android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -912,6 +924,13 @@ public void onDropViewInstance(RNCWebView view) {
         view.removeWebViewFromParent();
         RNCWebViewMapManager.INSTANCE.getRncWebViewMap().remove(webView.webViewKey);
         RNCWebViewMapManager.INSTANCE.getViewIdMap().remove(webView.getId());
+
+        if (view.temporaryParentNodeTag != 0) {
+          // Re-attach the internal webview to the temporary parent.
+          UIManagerModule uiManagerModule = ((ReactContext) view.getContext()).getNativeModule(UIManagerModule.class);
+          ViewGroup temporaryParentView = (ViewGroup)uiManagerModule.resolveView(view.temporaryParentNodeTag);
```

Comment:
I think `resolveView` may return null, so might want to add a null check just in case